src/index.ts

```diff
@@ -28,40 +28,52 @@
 //     unmarshalled from an incoming request.
 
 import * as minimist from 'minimist';
+
 import {
   ErrorHandler,
   FunctionSignatureType,
   getServer,
   getUserFunction,
 } from './invoker';
 
+// Supported command-line flags
+const FLAG = {
+  PORT: 'port',
+  TARGET: 'target',
+  SIGNATURE_TYPE: 'signature-type', // dash
+};
+
+// Supported environment variables
+const ENV = {
+  PORT: 'PORT',
+  TARGET: 'TARGET',
+  SIGNATURE_TYPE: 'SIGNATURE_TYPE', // underscore
+};
+
 enum NodeEnv {
   PRODUCTION = 'production',
 }
 
 const argv = minimist(process.argv, {
-  string: ['port', 'function-target', 'function-signature-type'],
+  string: [FLAG.PORT, FLAG.TARGET, FLAG.SIGNATURE_TYPE],
 });
 
 const CODE_LOCATION = process.cwd();
-const PORT = argv['port'] || process.env.PORT || '8080';
-const FUNCTION_TARGET =
-  argv['function-target'] || process.env.FUNCTION_TARGET || 'function';
+const PORT = argv[FLAG.PORT] || process.env[ENV.PORT] || '8080';
+const TARGET = argv[FLAG.TARGET] || process.env[ENV.TARGET] || 'function';
 
 const FUNCTION_SIGNATURE_TYPE_STRING =
-  argv['function-signature-type'] ||
-  process.env.FUNCTION_SIGNATURE_TYPE ||
-  'http';
+  argv[FLAG.SIGNATURE_TYPE] || process.env[ENV.SIGNATURE_TYPE] || 'http';
 const FUNCTION_SIGNATURE_TYPE =
```

Comment:
We could drop `FUNCTION_` prefix in this const as well, for consistency and brevity.

Reply:
Done.